.github/workflows/03_code_review.yml

```diff
@@ -42,6 +42,7 @@ jobs:
           project_key: ${{env.PROJECT_KEY}}
           coverage_exclusions: "**/config/*,**/*Mock*,**/model/**,**/entity/*"
           cpd_exclusions: "**/model/**,**/entity/*"
+          java_version: '17'
```

Comment:
set `uses: pagopa/github-actions-template/maven-code-review@v1.8.3` instead of `uses: pagopa/github-actions-template/maven-code-review@v1.4.1`